spec/requests/users_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe UsersController, type: :controller do
```

Comment:
- [ ] Please use `type: :request` while testing it should be like
```
 RSpec.describe "Users", type: :request do
   ...
 end
```

Reply:
@DeliceLydia I tried but it is not working